scripts/util/metadata/sink.rb

```diff
@@ -176,7 +176,7 @@ def initialize(hash)
           "unit" => "bytes"
         }
 
-      buffer_options["num_items"] =
+      buffer_options["max_events"] =
```

Comment:
@binarylogic I think this is all I have to change for this one?

Reply:
Yep, that's it.